src/components/date-input/index.md.njk

```diff
@@ -53,78 +53,97 @@ You will not normally need to use the `autocomplete` attribute in prototypes, as
 
 ### Error messages
 
-Error messages should be styled like this:
+Error messages should be styled like this if you’re highlighting the date as a whole:
```

Comment:
I can't think of a better way of expressing it.

Unless I've missed something, I think this is the first time we've made this concept explicit - ie you should use different error styles depending on whether just one element of the input is invalid, or the whole thing is invalid.

You could argue that we should cover that on the 'Error messages' page - but I think it makes most sense in the context of the date input, because that's the situation where it tends to come up.

Reply:
Thanks @StephenGill. I'm still not sure if the word "highlight" is clear, especially as the individual examples further down the page just say:
> Highlight the date input as a whole.

which doesn't explain what you need to do.

If we said: 
>[Show an error message](https://design-system.service.gov.uk/components/error-message/#how-it-works) for the date input as a whole.

and:

> Error messages should be styled like this if you’re [showing an error message](https://design-system.service.gov.uk/components/error-message/#how-it-works) for the date as a whole:

we could also link to the relevant guidance for how to do this. We already use the words "show an error message" in the [error message guidance](https://design-system.service.gov.uk/components/error-message/#when-to-use-this-component). 

What do you think? 🤔

Reply:
Thanks Hanna - let's go with Stephen's proposal for now, but keep an eye on Slack